src/libstore/store-api.hh

```diff
@@ -158,7 +158,9 @@ public:
 
     const Setting<int> pathInfoCacheSize{this, 65536, "path-info-cache-size", "size of the in-memory store path information cache"};
 
-    const Setting<bool> isTrusted{this, false, "trusted", "whether paths from this store can be used as substitutes even when they lack trusted signatures"};
+    const Setting<bool> isTrusted{this, false, "trusted", "whether paths from this store can be used as substitutes even when they lack trusted signatures. The dual of \"trusting\""};
```

Comment:
I think dual is a little too mathematical for user-facing text. How about just `see also`?

Reply:
Will do. FWIW, It isn't quite the true dual anyways, since it's the client's guess rather than server's response.